src/tests/Interop/LayoutClass/LayoutClassTest.cs

```diff
@@ -134,6 +134,9 @@ class StructureTests
         [DllImport("LayoutClassNative")]
         private static extern bool SimpleExpLayoutClassByRef(ExpClass p);
 
+        [DllImport("LayoutClassNative")]
+        private static extern bool SimpleBlittableSeqLayoutClass_Null(Blittable p);
```

Comment:
Should we add one for the non-blittable case too?

Reply:
I'll add one for completeness and regression testing.